src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts

```diff
@@ -0,0 +1,35 @@
+import { IPractice } from '../IPractice';
+import { PracticeEvaluationResult, PracticeImpact } from '../../model';
+import { DxPractice } from '../DxPracticeDecorator';
+import { PracticeContext } from '../../contexts/practice/PracticeContext';
+
+@DxPractice({
+  id: 'LanguageIndependent.ChangelogIsPresent',
+  name: 'Create a Changelog File',
+  impact: PracticeImpact.high,
+  suggestion:
+    'The main purpose of changelog is to show the differences, across multiple commits to communicate clearly to users.',
+  reportOnlyOnce: true,
+  url: 'https://dxkb.io/p/changelog',
```

Comment:
Article about changelog doesn't exist yet in our Knowledge Base. Please find a relevant source.

Reply:
Ok will do this change and resubmit

Reply:
Just this one should be fixed :)

Reply:
https://www.freecodecamp.org/news/a-beginners-guide-to-git-what-is-a-changelog-and-how-to-generate-it/

will this work ?

Reply:
Yes, this article seems fine to me :)

Reply:
I have updated the link for the article